Creating the Anvil Class Engine (#4594)
endless-sky/endless-sky: data/remnant outfits.txt

# Engines:

outfit "Anvil-Class Engine"
	category "Engines"
	licenses
		Remnant
	"cost" 290000
	thumbnail "outfit/tiny remnant engine"
	"mass" 26
	"outfit space" -26
	"engine capacity" -26
	"thrust" 13.5
	"thrusting energy" 2.65

> 	"thrusting heat" 2.5

Reviewer: The thrust energy and heat shouldn't be exactly the same as the Crucible-Class if the thrust is less. I'd suggest 2.05 energy and 1.95 heat.

Author: Alright. Changed

	"turn" 336.0
	"turning energy" 1.32

> 	"turning heat" 1.4

Reviewer: Again, the turning energy and heat shouldn't match the Crucible-Class if the turn is lesser. I'd suggest 1.05 energy and 1.1 heat.

Author: OK. Changed

	"flare sprite" "effect/remnant flare/small"
		"frame rate" 5
	"flare sound" "plasma tiny"
	description "This tiny set of engines were designed specifically for craft so small that they to combine both thrusters and steering into one unit."

outfit "Crucible-Class Thruster"
	category "Engines"
	licenses
		Remnant

> 	"cost" 190000

Reviewer: Why change the Crucible-Class thrusters?

Author: Sorry, that was accidental. Reverted.

# Engines:

outfit "Anvil-Class Engine"
	category "Engines"
	licenses
		Remnant
	"cost" 290000
	thumbnail "outfit/tiny remnant engine"
	"mass" 26
	"outfit space" -26
	"engine capacity" -26
	"thrust" 13.5
	"thrusting energy" 2.05
	"thrusting heat" 1.95
	"turn" 336.0
	"turning energy" 1.05
	"turning heat" 1.1
	"flare sprite" "effect/remnant flare/small"
		"frame rate" 5
	"flare sound" "plasma tiny"

> 	description "This tiny set of engines were designed specifically for craft so small that they to combine both thrusters and steering into one unit."

Reviewer: `that they to combine` Seems like you messed up here.
How about `"This tiny set of engines were designed specifically for craft so small that they're unable to fit a normal set of engines, leading to both thrusters and steering being combined into one unit."`

Author: Fixed by Pointedstick's change.

> 	description "This tiny set of engines were designed specifically for craft so small that they to combine both thrusters and steering into one unit."

Reviewer 2: ```suggestion
	description "This tiny set of engines was designed specifically for craft so small that it combines both thrusters and steering into one unit."
	
("set of engines" is a singular noun)
```
